frontend/src/lib/components/LemonActionableTooltip/LemonActionableTooltip.scss

```diff
@@ -18,17 +22,28 @@
         }
     }
     .LemonActionableTooltip__body {
-        padding: 0.5rem;
+        padding: 0.25rem 0.5rem;
+        > * + * {
+            margin-top: 0.5rem;
+        }
     }
     .LemonActionableTooltip__footer {
         display: flex;
         justify-content: space-between;
+        padding: 0.5rem;
     }
-    .LemonActionableTooltip__buttons {
+    .LemonActionableTooltip__url_buttons {
+        display: flex;
+        flex-direction: column;
+        > * + * {
+            margin-top: 0.5rem;
+        }
+    }
+    .LemonActionableTooltip__action_buttons {
```

Comment:
Nit: hyphens are usually used in places of spaces in BEM, they're a bit more readable than underscored